paas_theme/tables.py

```diff
@@ -61,21 +61,32 @@ class SearchResultTable(tables.Table):
         attrs={"td": {"class": "no-wrap"}},
     )
 
-    birth_place = tables.Column(accessor="place_of_birth", verbose_name="geboren in")
+    mitgliedschaft = tables.Column(
+        accessor="akademiemitgliedschaft",
+        verbose_name="Mitgieldschaft",
+        attrs={"td": {"class": "no-wrap"}},
```

Comment:
Why do we add `no-wrap` here? Having multi-line cells is actually better than a table that needs to be scrolled? Or is the `no-wrap` class doing something different